### DISCUSSION TITLE
Master clue step missing information about spade.

### DISCUSSION THREAD

**Discussion by anadia7** [runelite/runelite#15518] [Q&A] [answered]
The master clue step currently has a description of "North of Trollheim, where you fought Kamil for Desert Tresure." However there is no mention of digging or a spade like most clue steps have.

Just adding "Dig at" At the start would solve the confusion.

**Comment by Nightfirecat** [comment, marked as answer]
This clue already indicates if you do not have a spade, and the marked tile has a spade icon on it
![Screenshot_20220817_204656](https://user-images.githubusercontent.com/2199511/185288966-cc837ba0-8d7a-404b-affb-2169c444712c.png)
.

**Reply by geheur** [reply]
This particular clue is part of a master triple step clue which does not show that you need a spade.

**Reply by geheur** [reply]
![image](https://user-images.githubusercontent.com/41499327/185290070-e7fd124a-ac33-491c-9d2c-914ee18cf05e.png)
example of no spade requirement text.

**Reply by Nightfirecat** [reply]
I see--this is actually a duplicate of #15067 then, which #15068 sets out to solve.

**Reply by Nightfirecat** [reply]
I agree that the wording on this step was weird after having seen it ingame, so I've added 586a126ce44ba3effa9e207234a950fc9835ede0 to fix it.
